[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 100] Max packet in: 32768 bytes
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 100] Max packet out: 32768 bytes
[35mDEBUG   [0m paramiko.transport:transport.py:1893 Secsh channel 100 opened.
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 100] Sesch channel 100 request ok
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 100] EOF received (100)
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 100] EOF sent (100)
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 101] Max packet in: 32768 bytes
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 101] Max packet out: 32768 bytes
[35mDEBUG   [0m paramiko.transport:transport.py:1893 Secsh channel 101 opened.
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 101] Sesch channel 101 request ok
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 101] EOF received (101)
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 101] EOF sent (101)
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 102] Max packet in: 32768 bytes
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 102] Max packet out: 32768 bytes
[35mDEBUG   [0m paramiko.transport:transport.py:1893 Secsh channel 102 opened.
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 102] Sesch channel 102 request ok
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 102] EOF received (102)
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 102] EOF sent (102)
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 103] Max packet in: 32768 bytes
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 103] Max packet out: 32768 bytes
[35mDEBUG   [0m paramiko.transport:transport.py:1893 Secsh channel 103 opened.
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 103] Sesch channel 103 request ok
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 103] EOF received (103)
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 103] EOF sent (103)
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 104] Max packet in: 32768 bytes
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 104] Max packet out: 32768 bytes
[35mDEBUG   [0m paramiko.transport:transport.py:1893 Secsh channel 104 opened.
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 104] Sesch channel 104 request ok
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 104] EOF received (104)
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 104] EOF sent (104)
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 105] Max packet in: 32768 bytes
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 105] Max packet out: 32768 bytes
[35mDEBUG   [0m paramiko.transport:transport.py:1893 Secsh channel 105 opened.
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 105] Sesch channel 105 request ok
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 105] EOF received (105)
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 105] EOF sent (105)
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 106] Max packet in: 32768 bytes
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 106] Max packet out: 32768 bytes
[35mDEBUG   [0m paramiko.transport:transport.py:1893 Secsh channel 106 opened.
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 106] Sesch channel 106 request ok
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 106] EOF received (106)
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 106] EOF sent (106)
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 107] Max packet in: 32768 bytes
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 108] Max packet in: 32768 bytes
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 107] Max packet out: 32768 bytes
[35mDEBUG   [0m paramiko.transport:transport.py:1893 Secsh channel 107 opened.
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 108] Max packet out: 32768 bytes
[35mDEBUG   [0m paramiko.transport:transport.py:1893 Secsh channel 108 opened.
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 107] Sesch channel 107 request ok
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 108] Sesch channel 108 request ok
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 108] EOF received (108)
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 108] EOF sent (108)
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 107] EOF received (107)
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 107] EOF sent (107)
[35mDEBUG   [0m httpcore.connection:_trace.py:45 close.started
[35mDEBUG   [0m httpcore.connection:_trace.py:45 close.complete
[35mDEBUG   [0m httpcore.connection:_trace.py:45 connect_tcp.started host='192.168.0.207' port=1317 local_address=None timeout=10 socket_options=None
[35mDEBUG   [0m httpcore.connection:_trace.py:45 connect_tcp.complete return_value=<httpcore._backends.sync.SyncStream object at 0x7f3155cc9ea0>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_headers.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_headers.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_body.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_body.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_headers.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_headers.complete return_value=(b'HTTP/1.1', 404, b'Not Found', [(b'Content-Type', b'application/json'), (b'Grpc-Metadata-Content-Type', b'application/grpc+cosmos-sdk-grpc-codec'), (b'Grpc-Metadata-X-Cosmos-Block-Height', b'64787'), (b'X-Server-Time', b'1693275452'), (b'Date', b'Tue, 29 Aug 2023 02:17:32 GMT'), (b'Content-Length', b'114')])
[32mINFO    [0m httpx:_client.py:1013 HTTP Request: GET http://192.168.0.207:1317/cosmos/tx/v1beta1/txs/374B4CC32F874A44CFB5BD632C68A59DDA8E9F79922CE0E867C80BA9629533F1 "HTTP/1.1 404 Not Found"
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_body.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_body.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 response_closed.started
[35mDEBUG   [0m httpcore.http11:_trace.py:45 response_closed.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_headers.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_headers.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_body.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_body.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_headers.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_headers.complete return_value=(b'HTTP/1.1', 200, b'OK', [(b'Content-Type', b'application/json'), (b'Grpc-Metadata-Content-Type', b'application/grpc+cosmos-sdk-grpc-codec'), (b'Grpc-Metadata-X-Cosmos-Block-Height', b'64787'), (b'X-Server-Time', b'1693275452'), (b'Date', b'Tue, 29 Aug 2023 02:17:32 GMT'), (b'Transfer-Encoding', b'chunked')])
[32mINFO    [0m httpx:_client.py:1013 HTTP Request: GET http://192.168.0.207:1317/cosmos/tx/v1beta1/txs/0A53F8DF79BA796317AC68194DD86E4673F6A3DA38A87876EA7DF597AC17631A "HTTP/1.1 200 OK"
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_body.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_body.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 response_closed.started
[35mDEBUG   [0m httpcore.http11:_trace.py:45 response_closed.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_headers.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_headers.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_body.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_body.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_headers.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_headers.complete return_value=(b'HTTP/1.1', 404, b'Not Found', [(b'Content-Type', b'application/json'), (b'Grpc-Metadata-Content-Type', b'application/grpc+cosmos-sdk-grpc-codec'), (b'Grpc-Metadata-X-Cosmos-Block-Height', b'64788'), (b'X-Server-Time', b'1693275455'), (b'Date', b'Tue, 29 Aug 2023 02:17:35 GMT'), (b'Content-Length', b'114')])
[32mINFO    [0m httpx:_client.py:1013 HTTP Request: GET http://192.168.0.207:1317/cosmos/tx/v1beta1/txs/374B4CC32F874A44CFB5BD632C68A59DDA8E9F79922CE0E867C80BA9629533F1 "HTTP/1.1 404 Not Found"
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_body.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_body.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 response_closed.started
[35mDEBUG   [0m httpcore.http11:_trace.py:45 response_closed.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_headers.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_headers.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_body.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_body.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_headers.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_headers.complete return_value=(b'HTTP/1.1', 404, b'Not Found', [(b'Content-Type', b'application/json'), (b'Grpc-Metadata-Content-Type', b'application/grpc+cosmos-sdk-grpc-codec'), (b'Grpc-Metadata-X-Cosmos-Block-Height', b'64789'), (b'X-Server-Time', b'1693275458'), (b'Date', b'Tue, 29 Aug 2023 02:17:38 GMT'), (b'Content-Length', b'114')])
[32mINFO    [0m httpx:_client.py:1013 HTTP Request: GET http://192.168.0.207:1317/cosmos/tx/v1beta1/txs/374B4CC32F874A44CFB5BD632C68A59DDA8E9F79922CE0E867C80BA9629533F1 "HTTP/1.1 404 Not Found"
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_body.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_body.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 response_closed.started
[35mDEBUG   [0m httpcore.http11:_trace.py:45 response_closed.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_headers.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_headers.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_body.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_body.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_headers.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_headers.complete return_value=(b'HTTP/1.1', 404, b'Not Found', [(b'Content-Type', b'application/json'), (b'Grpc-Metadata-Content-Type', b'application/grpc+cosmos-sdk-grpc-codec'), (b'Grpc-Metadata-X-Cosmos-Block-Height', b'64789'), (b'X-Server-Time', b'1693275461'), (b'Date', b'Tue, 29 Aug 2023 02:17:41 GMT'), (b'Content-Length', b'114')])
[32mINFO    [0m httpx:_client.py:1013 HTTP Request: GET http://192.168.0.207:1317/cosmos/tx/v1beta1/txs/374B4CC32F874A44CFB5BD632C68A59DDA8E9F79922CE0E867C80BA9629533F1 "HTTP/1.1 404 Not Found"
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_body.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_body.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 response_closed.started
[35mDEBUG   [0m httpcore.http11:_trace.py:45 response_closed.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_headers.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_headers.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_body.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_body.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_headers.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_headers.complete return_value=(b'HTTP/1.1', 404, b'Not Found', [(b'Content-Type', b'application/json'), (b'Grpc-Metadata-Content-Type', b'application/grpc+cosmos-sdk-grpc-codec'), (b'Grpc-Metadata-X-Cosmos-Block-Height', b'64789'), (b'X-Server-Time', b'1693275464'), (b'Date', b'Tue, 29 Aug 2023 02:17:44 GMT'), (b'Content-Length', b'114')])
[32mINFO    [0m httpx:_client.py:1013 HTTP Request: GET http://192.168.0.207:1317/cosmos/tx/v1beta1/txs/374B4CC32F874A44CFB5BD632C68A59DDA8E9F79922CE0E867C80BA9629533F1 "HTTP/1.1 404 Not Found"
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_body.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_body.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 response_closed.started
[35mDEBUG   [0m httpcore.http11:_trace.py:45 response_closed.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_headers.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_headers.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_body.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_body.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_headers.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_headers.complete return_value=(b'HTTP/1.1', 404, b'Not Found', [(b'Content-Type', b'application/json'), (b'Grpc-Metadata-Content-Type', b'application/grpc+cosmos-sdk-grpc-codec'), (b'Grpc-Metadata-X-Cosmos-Block-Height', b'64789'), (b'X-Server-Time', b'1693275467'), (b'Date', b'Tue, 29 Aug 2023 02:17:47 GMT'), (b'Content-Length', b'114')])
[32mINFO    [0m httpx:_client.py:1013 HTTP Request: GET http://192.168.0.207:1317/cosmos/tx/v1beta1/txs/374B4CC32F874A44CFB5BD632C68A59DDA8E9F79922CE0E867C80BA9629533F1 "HTTP/1.1 404 Not Found"
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_body.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_body.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 response_closed.started
[35mDEBUG   [0m httpcore.http11:_trace.py:45 response_closed.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_headers.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_headers.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_body.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_body.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_headers.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_headers.complete return_value=(b'HTTP/1.1', 404, b'Not Found', [(b'Content-Type', b'application/json'), (b'Grpc-Metadata-Content-Type', b'application/grpc+cosmos-sdk-grpc-codec'), (b'Grpc-Metadata-X-Cosmos-Block-Height', b'64790'), (b'X-Server-Time', b'1693275470'), (b'Date', b'Tue, 29 Aug 2023 02:17:50 GMT'), (b'Content-Length', b'114')])
[32mINFO    [0m httpx:_client.py:1013 HTTP Request: GET http://192.168.0.207:1317/cosmos/tx/v1beta1/txs/374B4CC32F874A44CFB5BD632C68A59DDA8E9F79922CE0E867C80BA9629533F1 "HTTP/1.1 404 Not Found"
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_body.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_body.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 response_closed.started
[35mDEBUG   [0m httpcore.http11:_trace.py:45 response_closed.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_headers.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_headers.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_body.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_body.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_headers.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_headers.complete return_value=(b'HTTP/1.1', 404, b'Not Found', [(b'Content-Type', b'application/json'), (b'Grpc-Metadata-Content-Type', b'application/grpc+cosmos-sdk-grpc-codec'), (b'Grpc-Metadata-X-Cosmos-Block-Height', b'64790'), (b'X-Server-Time', b'1693275473'), (b'Date', b'Tue, 29 Aug 2023 02:17:53 GMT'), (b'Content-Length', b'114')])
[32mINFO    [0m httpx:_client.py:1013 HTTP Request: GET http://192.168.0.207:1317/cosmos/tx/v1beta1/txs/374B4CC32F874A44CFB5BD632C68A59DDA8E9F79922CE0E867C80BA9629533F1 "HTTP/1.1 404 Not Found"
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_body.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_body.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 response_closed.started
[35mDEBUG   [0m httpcore.http11:_trace.py:45 response_closed.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_headers.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_headers.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_body.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_body.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_headers.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_headers.complete return_value=(b'HTTP/1.1', 404, b'Not Found', [(b'Content-Type', b'application/json'), (b'Grpc-Metadata-Content-Type', b'application/grpc+cosmos-sdk-grpc-codec'), (b'Grpc-Metadata-X-Cosmos-Block-Height', b'64790'), (b'X-Server-Time', b'1693275476'), (b'Date', b'Tue, 29 Aug 2023 02:17:56 GMT'), (b'Content-Length', b'114')])
[32mINFO    [0m httpx:_client.py:1013 HTTP Request: GET http://192.168.0.207:1317/cosmos/tx/v1beta1/txs/374B4CC32F874A44CFB5BD632C68A59DDA8E9F79922CE0E867C80BA9629533F1 "HTTP/1.1 404 Not Found"
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_body.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_body.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 response_closed.started
[35mDEBUG   [0m httpcore.http11:_trace.py:45 response_closed.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_headers.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_headers.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_body.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_body.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_headers.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_headers.complete return_value=(b'HTTP/1.1', 404, b'Not Found', [(b'Content-Type', b'application/json'), (b'Grpc-Metadata-Content-Type', b'application/grpc+cosmos-sdk-grpc-codec'), (b'Grpc-Metadata-X-Cosmos-Block-Height', b'64791'), (b'X-Server-Time', b'1693275479'), (b'Date', b'Tue, 29 Aug 2023 02:17:59 GMT'), (b'Content-Length', b'114')])
[32mINFO    [0m httpx:_client.py:1013 HTTP Request: GET http://192.168.0.207:1317/cosmos/tx/v1beta1/txs/374B4CC32F874A44CFB5BD632C68A59DDA8E9F79922CE0E867C80BA9629533F1 "HTTP/1.1 404 Not Found"
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_body.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_body.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 response_closed.started
[35mDEBUG   [0m httpcore.http11:_trace.py:45 response_closed.complete
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 109] Max packet in: 32768 bytes
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 109] Max packet out: 32768 bytes
[35mDEBUG   [0m paramiko.transport:transport.py:1893 Secsh channel 109 opened.
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 109] Sesch channel 109 request ok
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 109] EOF received (109)
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 109] EOF sent (109)
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 110] Max packet in: 32768 bytes
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 110] Max packet out: 32768 bytes
[35mDEBUG   [0m paramiko.transport:transport.py:1893 Secsh channel 110 opened.
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 110] Sesch channel 110 request ok
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 110] EOF received (110)
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 110] EOF sent (110)